core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -45,11 +45,11 @@ import org.apache.kafka.common.requests.EpochEndOffset._
 import org.apache.kafka.common.requests.FetchRequest.PartitionData
 import org.apache.kafka.common.requests.FetchResponse.AbortedTransaction
 import org.apache.kafka.common.requests.ProduceResponse.PartitionResponse
-import org.apache.kafka.common.requests._
+import org.apache.kafka.common.requests.{ApiError, DeleteRecordsResponse, DescribeLogDirsResponse, EpochEndOffset, IsolationLevel, LeaderAndIsrRequest, LeaderAndIsrResponse, OffsetsForLeaderEpochRequest, StopReplicaRequest, UpdateMetadataRequest}
```

Comment:
I find that one line imports like `some.package.{One, Two, Three}` lead to unnecessary merge conflicts. How about encouraging one line imports like:
```
import some.package.One
import some.package.Two
...
```

Reply:
This is the current style. If we want to change it, let's start a discussion in the mailing list.